pages/roles-and-teams.md

```diff
@@ -2,60 +2,21 @@
 
-### Comms Team
-
-**Director of Outreach**
-
-The Director of Outreach drives a comprehensive internal and external communications strategy for 18F. This includes editorial and content on our website, social media, events/conferences, messaging, and general community evangelism.
+### Communications Team
+[**Director of Outreach**](https://pages.18f.gov/joining-18f/roles-and-teams/positions/director-of-outreach/)
 
 ### Consulting Team
+**Design and Product Strategist** *(Role profile coming soon)*
 
-**Acquisition Specialist**
-
-Acquisition specialists help teams support digital services through procurement efforts, investigate other agencies’ tools and processes to find opportunities to improve results, and work with contracting officers to help procure goods and services for 18F and 18F partner agencies.
-
-**Data Architect**
-
-Data architects on the 18F consulting team help agencies liberate and embrace their data, equip them with profound insights drawn from that data, empowering them to more effectively fulfill their mission.
-
-**Practical Dev**
-
-Practical devs on the 18F consulting team provide broad development expertise and use their consultative know-how to help agencies by conducting workshops and protosketching sessions to build out proofs-of-concept and prototypes.
+The Design and Product Strategist is at the forefront of human-centered design and culture change within the federal government. As a consultant, you’ll affect how agencies, programs, and individuals perceive and examine problems, dream of solutions, and find innovative ways to make those solutions a successful reality. Depending on the methods you use, you can effect change in a few months (or even days).
 
 ### Delivery Team
+**Site Reliability Engineer** *(Role profile coming soon)*
 
-**DevOps & Site Reliability Engineer**
-
-DevOps engineers work tirelessly to arm developers with the best tools and ensure system uptime and performance, helping to bridge gaps with legacy development or operations teams.
-
-**Product Lead**
-
-18F product leads are responsible for managing the development, delivery, and continuous improvement of products while establishing and managing our relationship with our partners (clients).
-
-**Software Developer (Front-End, Back-End, Full-Stack)**
+Site Reliability Engineers ensure our platforms and infrastructure systems &mdash; and the applications on top of them &mdash;  are reliable and easy to deploy and maintain. You’ll be involved in designing and implementing systems in a human-centered way, communicating with users, and finding solutions to help people work more smoothly. Along with providing the platform, you seek opportunities and get creative in  architecting solutions.
```

Comment:
@rooneywp The singular vs. plural suggests that we're hiring for *one* design and product strategist but *multiple* SREs. Is that accurate?